Week2/joyeeon/Mission2/src/components/Navbar.tsx

```diff
@@ -9,7 +9,7 @@ const Navbar = () =>{
 
     return (
         <nav className={`
-            w-full h-12 flex items-center justify-between px-10
+            w-full h-12 flex items-center justif px-10
```

Comment:
이 부분에 오타가 난 것 같습니다...!!
```suggestion
            w-full h-12 flex items-center justify-between px-10
```